d-mode.el

```diff
@@ -715,12 +716,8 @@ The expression is added to `compilation-error-regexp-alist' and
   :type 'hook
   :group 'c)
 
-;; For compatibility with Emacs < 24
-(defalias 'd-parent-mode
-  (if (fboundp 'prog-mode) 'prog-mode 'fundamental-mode))
-
```

Comment:
This is what was confusing `minimap-mode` in #86. The parent mode of `d-mode` turns out to be `d-parent-mode`, not `prog-mode`, i.e. the `defalias` is not transparent (unlike a D `alias`, heh).